Resources/public/js/ezrecommendation.js

```diff
@@ -0,0 +1,77 @@
+// register HandleBars helpers when library is loaded
+
+document.addEventListener('DOMContentLoaded', function() {
```

Comment:
Nitpick: do you know what is the difference between `DOMContentLoaded` and `load` events?